examples/go/hello/hello.go

```diff
@@ -46,9 +41,8 @@ func main() {
 	// Serves the icon for the app.
 	http.HandleFunc("/static/icon.png", writeData("image/png", iconData))
 
-	addr := fmt.Sprintf("%v:%v", host, port)
-	fmt.Printf(`hello-world app listening at http://%s`, addr)
-	http.ListenAndServe(addr, nil)
+	fmt.Println("Listening on :8080") // matches manifest.json
+	panic(http.ListenAndServe(":8080", nil))
```

Comment:
Why `panic`?

Reply:
ListenAndServe is a blocking call, unless it returns a Listen error immediately. Can replace it with `log.Fatal`?